.claude/skills/corpus-test/SKILL.md

```
@@ -0,0 +1,75 @@
+If no corpus files found, download them:
+
+```bash
+make corpus-download
+```
+
+## Step 2: Run Corpus Tests
+
+Based on the mode argument:
+
+**Full mode** (default):
+```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)